spec/features/landing_face_to_face_filter_for_other_advice_types_spec.rb

```diff
@@ -19,19 +19,15 @@
 
   def given_firms_with_advisers_were_previously_indexed
     with_fresh_index! do
-      @first = create(:firm_with_no_business_split, retirement_income_products_flag: true, wills_and_probate_flag: true, other_flag: true)
+      @first = create(:firm_with_no_business_split, retirement_income_products_flag: true, wills_and_probate_flag: true)
       @leicester = create(:adviser, firm: @first, latitude: 52.633013, longitude: -1.131257)
 
-      @second = create(:firm_with_no_business_split, retirement_income_products_flag: true, wills_and_probate_flag: true, other_flag: true)
+      @second = create(:firm_with_no_business_split, retirement_income_products_flag: true, wills_and_probate_flag: true)
       @glasgow = create(:adviser, firm: @second, latitude: 55.856191, longitude: -4.247082)
 
-      @excluded = create(:firm_with_no_business_split, retirement_income_products_flag: true, other_flag: true)
+      @excluded = create(:firm_with_no_business_split, retirement_income_products_flag: true)
       create(:adviser, firm: @excluded, latitude: 51.428473, longitude: -0.943616)
 
-      create(:firm_with_no_business_split, other_flag: true) do |f|
-        create(:adviser, firm: f, latitude: 51.428473, longitude: -0.943616)
-      end
-
```

Comment:
If these are needed like I think they are, then lets put comments by them to prevent future confusion about their purpose.